components/properties/src/bidi_data.rs

```diff
@@ -0,0 +1,137 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use zerovec::ule::{AsULE, CharULE, ULE};
+use zerovec::ZeroVecError;
+
+use crate::props::{BidiPairedBracketType, BidiPairedBracketTypeULE};
+
+/// TODO: This module's description
+///
+/// This module provides an efficient storage of data serving the following
+/// properties:
+/// - `Bidi_Paired_Bracket`
+/// - `Bidi_Paired_Bracket_Type`
+/// - `Bidi_Mirrored`
+/// - `Bidi_Mirroring_Glyph`
+///
+
+
+/// 20..0  Code point return value for Bidi_Mirroring_Glyph value
```

Comment:
these docs should go on the ULE type

Also, this module should probably be a submodule of `provider` and marked unstable the way we usually do provider code, but not `doc(hidden)`

Reply:
also these ranges seem backwards?